Bio/PDB/__init__.py

```diff
@@ -31,6 +31,8 @@
 # IO of PDB files (including flexible selective output)
 from .PDBIO import PDBIO, Select
 
+from .MMCIFIO import MMCIFIO
```

Comment:
Is this needed? The ``Bio.PDB`` module already does too many automatic imports (and name shadowing) so I'd prefer to avoid it.

Reply:
Yes I will remove this.